README.md

```diff
@@ -20,39 +21,58 @@ and configure these applications.
   - [Example DataScienceCluster](#example-datasciencecluster)
   - [Run functional Tests](#run-functional-tests)
   - [Run e2e Tests](#run-e2e-tests)
+  - [API Overview](#api-overview)
+  - [Component Integration](#component-integration)
   - [Troubleshooting](#troubleshooting)
+  - [Upgrade testing](#upgrade-testing)
 
 ## Usage
 
+### Prerequisites
+If `single model serving configuration` is used or if `Kserve` component is used then please make sure to install the following operators before proceeding to create a DSCI and DSC instances.
+ - [Authorino operator](https://github.com/Kuadrant/authorino)
+ - [Service Mesh operator](https://github.com/Maistra/istio-operator)
+ - [Serverless operator](https://github.com/openshift-knative/serverless-operator)
+
+Additionally installing `Authorino operator` & `Service Mesh operator` enhances user-experience by providing a single sign on experience.
+
 ### Installation
 
-The latest version of operator can be installed from the `community-operators` catalog on `OperatorHub`. It can also be build
+- The latest version of operator can be installed from the `community-operators` catalog on `OperatorHub`.
+
+  ![ODH operator in OperatorHub](docs/images/OperatorHub%20ODH%20Operator.png)
+
+  Please note that the latest releases are made in the `Fast` channel.
+
+- It can also be build
 and installed from source manually, see the Developer guide for further instructions.
 
-1. Subscribe to operator by creating following subscription
+  1. Subscribe to operator by creating following subscription
 
-    ```console
-    cat <<EOF | oc create -f -
-    apiVersion: operators.coreos.com/v1alpha1
-    kind: Subscription
-    metadata:
-      name: opendatahub-operator
-      namespace: openshift-operators
-    spec:
-      channel: fast
-      name: opendatahub-operator
-      source: community-operators
-      sourceNamespace: openshift-marketplace
-    EOF
-    ```
+      ```console
+      cat <<EOF | oc create -f -
+      apiVersion: operators.coreos.com/v1alpha1
+      kind: Subscription
+      metadata:
+        name: opendatahub-operator
+        namespace: openshift-operators
+      spec:
+        channel: fast
+        name: opendatahub-operator
+        source: community-operators
+        sourceNamespace: openshift-marketplace
+      EOF
+      ```
 
-2. Create [DSCInitializationc](#example-dscinitialization) CR manually.
-  You can also use operator to create default DSCI CR by removing env variable DISABLE_DSC_CONFIG from CSV following restart operator pod.
+  2. Create [DSCInitialization](#example-dscinitialization) CR manually.
+    You can also use operator to create default DSCI CR by removing env variable DISABLE_DSC_CONFIG from CSV or changing the value to "false", followed by restarting the operator pod.
 
-3. Create [DataScienceCluster](#example-datasciencecluster) CR to enable components
+  3. Create [DataScienceCluster](#example-datasciencecluster) CR to enable components
 
 ## Dev Preview
 
```

Comment:
We can get rid of Dev Preview section and dev-preview.md. It was created when we did not have odh releases for v2